R/FilterState.R

```diff
@@ -281,74 +281,80 @@ FilterState <- R6::R6Class( # nolint
     ui = function(id, parent_id = "cards") {
       ns <- NS(id)
 
-      tags$div(
+      # Filter card consists of header and body, arranged in a single column.
+      # Body is hidden and is toggled by clicking on header.
+      ## Header consists of title and summary, arranged in a column.
+      ### Title consists of conditional icon, varname, conditional varlabel, and controls, arranged in a row.
+      ### Summary consists of value and controls, arranged in a row.
```

Comment:
```suggestion
      # Filter card consists of header and body, arranged in a single column.
      # Body is hidden and is toggled by clicking on header.
      # Header consists of title and summary, arranged in a column.
      # Title consists of conditional icon, varname, conditional varlabel, and controls, arranged in a row.
      # Summary consists of value and controls, arranged in a row.
```

Reply:
Could you kindly review this, @chlebowa?

Reply:
This is done on purpose to improve readability. I tried to use indention but the bot removed it.